### ISSUE TITLE
C++Math.c compile error with 2.12.2, Xcode 12.5.1, xcode-select version 2384

### ISSUE THREAD

**Issue by mazz** [LeoNatan/LNPopupController#474]
Compilation error 2.12.2:

```
/Users/michael/Library/Developer/Xcode/DerivedData/RVG-epmdhrvwdpajjwhlhlnjsmumaifo/SourcePackages/checkouts/LNPopupController/LNPopupController/LNPopupController/Private/C++Math.cc:16:14: error: no member named 'clamp' in namespace 'std'
        return std::clamp(v, lo, hi);
               ~~~~~^
1 error generated.
```

```
➜  kjvrvg-ios git:(feature/1.2-legacy) ✗ xcode-select -v       
xcode-select version 2384.
```

**To Reproduce**
1. Swift Package install LNPopupController 2.12.2, Xcode 12.5.1, xcode-select version 2384
2. Compile
4. error above

**Expected Behavior**
no compilation error

**Comment by LeoNatan** [comment]
Hmmm, when building through Xcode 12.5 directly, it succeeds.
Perhaps C++17 isn't available on Xcode when using a package? 🤦‍♂️

**Comment by mazz** [comment]
My current workaround is to set the version to 2.10.37 Up to next Major Version.
<img width="505" alt="Screen Shot 2021-08-11 at 11 49 56 AM" src="https://user-images.githubusercontent.com/808934/129061774-92e5b673-d95c-45d0-b9a6-0b972c7f82ce.png">

**Comment by LeoNatan** [comment]
I will release a fix soon. I'll just reimplement the clamp function internally.

**Comment by LeoNatan** [comment]
Fixed in 2.12.3
